src/utils.js

```diff
@@ -0,0 +1,130 @@
+            numEtraction--
+        }
+        return [...numbersArry].join(',')
+    }
+}
+
+/**
+ * func to determine how many bills the user want to generate
+ * @returns {Number} number of bills
+ */
+// 
+const howManyBills = () =>{
+    const minNum = 0
+    const maxNum = 5
+    let userNumber = parseInt(prompt('Please enter the number of bills you want to play between 1 and 5 (0 to exit): ',0))
+
+    while((userNumber<minNum || userNumber>maxNum) || isNaN(userNumber)){
+        userNumber = +parseInt(prompt('Please enter a number between 1 and 5 (0 to exit): ',0))
+    }
+    if (userNumber===0){console.log(`the request has been stoped.`)}
+    return userNumber
+}
+
+/**
+ * func to initializate the bills
+ * @param {Number} billsNumber 
+ * @returns {Array of Object}
+ */
+const initializeUserBills = billsNumber => {
+    const billsWithContent = []
+    if (isNaN(billsNumber)||billsNumber===0){ console.log('There was no initialization of bills.') }
+    else{
+        for(i=1;i<=billsNumber;i++){
+            billsWithContent.push(new Bill(i))
+        }
+        return billsWithContent
+    }
+}
+
+/**
+ * func to print bills
+ * @param {Array of Object} bills 
+ * @returns {String} console.log the final result in a nice template
+ */
+const printBills = (bills) => {
+
+if(bills === undefined){return  `Please restart the software and select the number of bills you want to play.`}
+
+//max number of characters consider in the print in each sections
+const maxBillNChar = 12 
+const maxCityChar = 10
+const maxTypeChar = 10
+const maxNumbersChar = 31
+
+const header = `
++------------+----------+----------+-------------------------------+
+¦   Bill n°  ¦   City   ¦   Type   ¦            Numbers            ¦
++------------+----------+----------+-------------------------------+`
+const footer = `+------------+----------+----------+-------------------------------+`
+
+const arrayOfString = []
+bills.forEach(bill => {
+
+    const billNameString =`¦` + bill['name'].toString().padStart(maxBillNChar/2,' ') + `¦`.padStart((maxBillNChar/2)+1,' ')
+    const cityNameString = bill['city'].toString().padStart(bill['city'].toString().length + 1,' ') + `¦`.padStart(maxCityChar-bill['city'].toString().length,' ')
+    const typeNameString = bill['type'].toString().padStart(bill['type'].toString().length + 1,' ') + `¦`.padStart(maxTypeChar-bill['type'].toString().length,' ')
+    const numbersNameString = bill['numbers'].toString().padStart(bill['numbers'].toString().length + 1,' ') + `¦`.padStart(maxNumbersChar-bill['numbers'].toString().length,' ')
+
+    const totalString = billNameString + cityNameString + typeNameString + numbersNameString
+    arrayOfString.push(`${totalString}`)
+});
+output = `${header}\n${arrayOfString.join('\n')}\n${footer}`
+return output
+}
+
+exports.module = {Bill, howManyBills, initializeUserBills, printBills}
+
+//personal test
+console.log(printBills(initializeUserBills(howManyBills())))
```

Comment:
For calling main functions use **index.js** file (root directory) for support functions **utils.js**